eZ/Publish/Core/Persistence/Legacy/Content/Gateway/DoctrineDatabase.php

```diff
@@ -1534,10 +1534,57 @@ public function setName($contentId, $version, $name, $language)
             $this->dbHandler->quoteColumn('name'),
             $q->bindValue($name)
         );
+        $q->bindValue($language->id, ':languageId', \PDO::PARAM_INT);
+        $q->bindValue($contentId, ':contentId', \PDO::PARAM_INT);
         $q->prepare()->execute();
     }
 
     /**
+     * Returns a language sub select query for setName.
+     *
```

Comment:
Would like some explanation here on why this is needed, and what the function actually does _(it's hard to read the code here)_. Maybe also better reflect what this does in the function name.